core/breakpoints/manager.php

```diff
@@ -0,0 +1,330 @@
+		];
+
+		return $values_map[ $name ];
+	}
+
+	/**
+	 * Get Config
+	 *
+	 * Retrieve --Enabled-- Breakpoints Config
+	 *
+	 * @since 3.2.0
+	 *
+	 * @param string|null $breakpoint_name
+	 * @return array All breakpoints config, or a specific breakpoint's config.
+	 */
+	public function get_config( $breakpoint_name = null ) {
+		if ( ! $this->config ) {
+			$this->config = $this->init_config();
+		}
+
+		return self::get_items( $this->config, $breakpoint_name );
+	}
+
+	/**
+	 * Init Config
+	 *
+	 * Iterate over the breakpoint instances to create the breakpoints config array.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return array $config
+	 */
+	private function init_config() {
+		$config = [];
+
+		// Make sure breakpoint instances are initialized before fetching their config.
+		$breakpoints = $this->get_breakpoints();
+
+		// Iterate over the breakpoint instances and get each one's config.
+		foreach ( $breakpoints as $name => $instance ) {
+			/** @var Breakpoint $instance */
+			$breakpoint_config = $instance->get_config();
+
+			// Only add breakpoints to the config array if they are enabled.
+			if ( $breakpoint_config['is_enabled'] ) {
+				$config[ $name ] = $breakpoint_config;
+			}
+		}
+
+		return $config;
+	}
+
+	/** Has Custom Breakpoints
+	 *
+	 * Checks whether there are currently custom breakpoints saved in the database.
+	 * Returns true if there are breakpoint values saved in the active kit.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return boolean
+	 */
+	public function has_custom_breakpoints() {
+		$kit_settings = Plugin::$instance->kits_manager->get_current_settings();
+		$breakpoint_names = array_keys( self::get_default_config() );
+
+		$has_custom_breakpoints = false;
+
+		foreach ( $breakpoint_names as $breakpoint_name ) {
+			if ( ! empty( $kit_settings[ $breakpoint_name ] ) ) {
+				$has_custom_breakpoints = true;
+				break;
+			}
+		}
+
+		return $has_custom_breakpoints;
+	}
+
+	/**
+	 * Get Stylesheet Templates Path
+	 *
+	 * @since 3.2.0
+	 * @access public
+	 * @static
+	 */
+	public static function get_stylesheet_templates_path() {
+		return ELEMENTOR_ASSETS_PATH . 'css/templates/';
+	}
+
+	/**
+	 * Compile Stylesheet Templates
+	 *
+	 * @since 3.2.0
+	 * @access public
+	 * @static
+	 */
+	public static function compile_stylesheet_templates() {
+		foreach ( self::get_stylesheet_templates() as $file_name => $template_path ) {
+			$file = new Frontend( $file_name, $template_path );
+
+			$file->update();
+		}
+	}
+
+	/**
+	 * Get Stylesheet Templates
+	 *
+	 * @since 3.2.0
+	 * @access private
+	 * @static
+	 */
+	private static function get_stylesheet_templates() {
+		$templates_paths = glob( self::get_stylesheet_templates_path() . '*.css' );
+
+		$templates = [];
+
+		foreach ( $templates_paths as $template_path ) {
+			$file_name = 'custom-' . basename( $template_path );
+
+			$templates[ $file_name ] = $template_path;
+		}
+
+		return apply_filters( 'elementor/core/responsive/get_stylesheet_templates', $templates );
```

Comment:
Deprecate this filter and create a new one `elementor/core/breakpoints/get_stylesheet_template`

Reply:
Done.